core/src/main/kotlin/team/backend/exception/CommonExceptionHandler.kt

```diff
@@ -0,0 +1,39 @@
+package team.backend.exception
+
+import org.springframework.http.ResponseEntity
+import org.springframework.validation.FieldError
+import org.springframework.web.bind.annotation.ExceptionHandler
+import org.springframework.web.bind.annotation.RestControllerAdvice
+import org.springframework.web.bind.support.WebExchangeBindException
+import reactor.core.publisher.Mono
+import team.backend.response.Error
+import team.backend.response.FailResponse
+import team.backend.response.ResponseCode
+
+@RestControllerAdvice
+class CommonExceptionHandler {
```

Comment:
__Order:__

- At the top, child exceptions
- At the bottom, Exception.class

> child -> parent

Reply:
@BAEKJungHo 
Is there any specical reason?
e.g. clean code, mechanism..

Reply:
@BAEKJungHo 
I applied.